sdk/communication/communication-short-codes/test/public/utils/recordedClient.ts

```diff
@@ -76,7 +74,7 @@ export function createRecordedClientWithToken(
   context: Context
 ): RecordedClient<ShortCodesClient> | undefined {
   const recorder = record(context, environmentSetup);
-  let credential: TokenCredential;
```

Comment:
what's the reason for getting rid of the `:TokenCredential`?

Reply:
No specific reason, it was just a small oversight. Thanks for pointing it out!